tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -0,0 +1,152 @@
+              Write-Host "git clone https://github.com/Microsoft/vcpkg"
+              git clone https://github.com/Microsoft/vcpkg
+              if ($LASTEXITCODE -ne 0) {
+                Write-Error "Unable to check out vcpkg fork repo"
+                exit $LASTEXITCODE
+              }
+              cd ./vcpkg
+              git fetch --tags
+              git checkout tags/$(VcpkgRelease)
+            displayName: Clone vcpkg 
+            workingDirectory: '$(Build.SourcesDirectory)'
+          - pwsh: |
+                ./vcpkg/bootstrap-vcpkg
+            displayName: Bootstrap vcpkg.
+            condition: succeeded()
+            workingDirectory: '$(Build.SourcesDirectory)'
+          - pwsh: |
+              $vcpkgRoot = Resolve-Path "./vcpkg"
+              Write-Host "Set VCPKG_ROOT: $vcpkgRoot"
+              Write-Host "##vso[task.setvariable variable=VCPKG_ROOT]$vcpkgRoot"
+            displayName: Set VCPKG_ROOT
+            condition: succeeded()
+          - pwsh: mkdir build
+            displayName: Create cmake build directory.
+            workingDirectory: '$(Build.SourcesDirectory)/tools/apiview/parsers/cpp-api-parser'
+            condition: succeeded()
+          - task: PowerShell@2
+            inputs:
+              pwsh: true
+              targetType: filePath
+              filePath: eng/scripts/Set-VcpkgWriteModeCache.ps1
+              arguments: -StorageAccountKey '$(cpp-vcpkg-cache-storage-key)'
+            displayName: Set Vcpkg Write-mode Cache
+          - pwsh: |
+              cmake.exe -G "Visual Studio 16 2019" -DCMAKE_CXX_STANDARD:STRING="20" -DCMAKE_TOOLCHAIN_FILE:STRING=${env:VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake -DVCPKG_BUILD_TYPE:STRING="release" -DVCPKG_OVERLAY_TRIPLETS:STRING="../vcpkg-triplets" -DMSVC_RUNTIME_LIBRARY:STRING="MultiThreadedDebug" -DVCPKG_TARGET_TRIPLET:STRING="x64-windows-static-release" ..
+            displayName: CMAKE Generate.
+            workingDirectory: $(Build.SourcesDirectory)/tools/apiview/parsers/cpp-api-parser/build
+            condition: succeeded()
+            env:
+              VCPKG_BINARY_SOURCES: $(VCPKG_BINARY_SOURCES_SECRET)
+          - pwsh: |
+              cmake.exe --build . --parallel 8 --target ParseAzureSdkCpp --clean-first --config Release
+            workingDirectory: $(Build.SourcesDirectory)/tools/apiview/parsers/cpp-api-parser/build
+            displayName: Build ApiView Parser.
+            condition: succeeded()
+          - task: UniversalPackages@0
+            displayName: Universal Publish
+            inputs:
+              command: publish
+              publishDirectory: $(Build.SourcesDirectory)/tools/apiview/parsers/cpp-api-parser/build/ParseAzureSdkCpp
+              vstsFeedPublish: 'AzureSDKForCpp '
+              vstsFeedPackagePublish: 'test-universal-package'
+              versionOption: minor
+              packagePublishDescription: Azure C++ SDK API View Parser.
+            condition: succeeded()
```

Comment:
`condition: succeeded()` is the default condition. 

Releasing from PRs is not a good practice so the condition here should skip the step if it's a PR run. We can publish instead to the pipeline's artifacts which will make the bits from the build available for scrutiny. 

Zooming out, we need to figure out how we're releasing this package. A couple of ways to do this: 

1. Publish on merges changing relevant files in the `main` branch -- In this case you get faster and less manual turnaround on updates but you need to be confident that the bits you're releasing work. 
2. Publishing from manual runs -- This requires some manual work to release but prevents releases of various

Reply:
Changed the publish step to be manual.